cogs5e/models/automation/effects/usecounter.py

```diff
@@ -77,7 +73,12 @@ def run(self, autoctx):
         autoctx.metavars['lastCounterUsedAmount'] = result.used_amount
         return result
 
-    def get_and_use_counter(self, autoctx, amount):  # this is not in run() because indentation
+    def get_and_use_counter(self, autoctx, amount, ignore_resources: bool = False):  # this is not in run() because indentation
+        if ignore_resources:
+            return UseCounterResult(counter_name=self.counter.name,
```

Comment:
This is an AttributeError since the type of `self.counter` here is `str | AbilityReference`